scripts/upgradeConfig.ts

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+import { readdir, readFile, writeFile } from 'fs-extra';
+import simpleGit from 'simple-git';
+
+/**
+ * Script: upgradeConfigs.ts
+ * Run with: ts-node scripts/upgradeConfig.ts
+ * ------------------------------------------
+ * Upgrades the references to \@solid/community-server in
+ * package.json (lsd prefix) and all JSON-LD config files.
+ * This script is run alongside standard-version
+ * after the version bump is done in package.json but before the
+ * release has been committed.
+ */
+
+/**
+ * Helper function to escape search strings for use in Regular Expressions
+ * @param  string - the search string to be escaped
+ * @returns Promise with the escaped string
+ */
+async function escapeString(string: string): Promise<string> {
+  return string.replace(/[$()*+.?[\\\]^{|}]/gu, '\\$&');
+}
+
+interface NpmPackage {
+  version: string;
+  name: string;
+}
+/**
+ * Reads and return the package.json file to extract version and name
+ * @returns Promise with the NpmPackage
+ */
+async function getNpmPackage(): Promise<NpmPackage> {
+  return JSON.parse(await readFile('package.json', 'utf8'));
+}
+/**
+ * Search and replace the version of a component with given name
+ * @param  filePath - File to search/replace
+ * @param  name - Component name
+ * @param  version - Semantic version to change to
+ */
+async function replaceComponentVersion(filePath: string, name: string, version: string): Promise<void> {
+  console.log(`Replacing version in ${filePath}`);
+  const data = await readFile(filePath, 'utf8');
+  const escapedName = await escapeString(name);
+  const regex = new RegExp(`(${escapedName}\\/)\\^\\d+\\.\\d+\\.\\d+`, 'gmu');
```

Comment:
Would let this function take a regex as input since it's going to be the same for all calls.

Reply:
Regex can also be rewritten like this to include less backslashes which might help readability.
```suggestion
  const regex = new RegExp(`(${escapedName}/)${/\^\d+\.\d+\.\d+/.source}`, 'gmu');
```